examples/import-json.php

```diff
@@ -1,19 +0,0 @@
-<?php
-
-$file = "http://media.mongodb.org/zips.json";
-
-$zips = file($file, FILE_IGNORE_NEW_LINES);
-
-
-$batch = new MongoDB\WriteBatch(true);
```

Comment:
The original plan was to have consistent dataset for all examples, queries and output, in the docs.